sharedbuffers/mapped_struct.py

```diff
@@ -966,9 +968,10 @@ def __iter__(self):
 
     def __reversed__(self):
         l = len(self)
-        if l > 0:
-            for i in xrange(l - 1, -1, -1):
-                yield self[i]
+        if l == 0:
+            return self._make_empty()
```

Comment:
You can't return a value from a generator.

You can however stop iteration, by just `return`ing.

Reply:
Ah, right. I think I'll just emulate what happens with native lists and tuples.